src/vm/disassembler.cpp

```diff
@@ -78,16 +77,49 @@ void Disassembler::StaticInitialize()
 #if USE_COREDISTOOLS_DISASSEMBLER
     _ASSERTE(!IsAvailable());
 
-    // TODO: The 'coredistools' library will eventually be part of a NuGet package, need to be able to load
-    // that using appropriate search paths
-    LPCWSTR libraryName = MAKEDLLNAME(W("coredistools"));
-    HMODULE libraryHandle = CLRLoadLibrary(libraryName);
+	LPCWSTR libraryName = MAKEDLLNAME(W("coredistools"));
```

Comment:
It looks like one of editors on my machines doesn't get configured to use spaces.. I will fix this.